examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,6 +1193,85 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+static void snapshot_examples(mongocxx::client& client) {
+    // Write concern "majority" needed to avoid the following error below:
+    //
+    // Unable to read from a snapshot due to pending collection catalog changes;
+    // please retry the operation.
+    //
+    // See also: https://jira.mongodb.org/browse/SERVER-41532
+
+    // Start Example 59
```

Comment:
```suggestion
    // Start Snapshot Query Example 1
```

I missed this in the C review. These comments are scraped by the docs team. I think they need to be an exact match of what was requested in the ticket.

Reply:
Got it! How about I also update the C driver example accordingly.